client/src/pages/Dialogs/EditListDialog.js

```diff
@@ -33,24 +34,27 @@ function EditListDialog(props) {
     );
   };
 
-  fetch("/itemLists/getProductList/?listName=" + listName, {
-    method: "GET",
-    headers: {
-      "Content-Type": "application/json",
-    },
-  })
-    .then((res) => {
-      if (res.status === 200) {
-        return res.json();
-      }
+  if (!productListRetrieved) {
```

Comment:
Wait, I thought fixed this in the branch that I implemented it in already. I'm guessing it didn't go through?